src/app/pages/news/11.3/01-variables-editor.md

```diff
@@ -0,0 +1,14 @@
+## Variables Editor (Alpha) 🧪 {#variablesEditor}
+
+The first of many next-generation config editors has arrived!
+
+- **Low-Code**: Instead of having to navigate a YAML document that can get pretty big, you can now edit your variables with a handy user interface.
+- **Features**: Add or delete variables, and find what you are looking for by using the integrated search functionality. After selecting a variable, you can see all its information in the detail view. Change its name, value, description, and metadata while the UI adapts to your choices.
+- **Alpha**: More features and improvements to the user experience are coming.
```

Comment:
not sure about the 'alpha' flag on this editor.
sure it's a first version, and we will continue improving it.
but its already much closer to its final featureset than other 'alphas' like the 'form-editor' or the 'web-ide'. 
maybe we should just call it 'beta'. What do you think @weissreto

Reply:
Yeah, it is not an alpha anymore. I would also not call it a 'beta'. Sure we will improve it further. However, for the user, it is now much easier than editing the raw yaml file. 
@ivy-lgi be proud of your work, and call it production-ready.